src/server/game/Server/Packets/NPCPackets.h

```diff
@@ -26,6 +26,29 @@ namespace WorldPackets
 {
     namespace NPC
     {
+        class CloseInteraction final : public ClientPacket
+        {
+        public:
+            CloseInteraction(WorldPacket&& packet) : ClientPacket(CMSG_CLOSE_INTERACTION, std::move(packet)) { }
+
+            void Read() override;
+
+            ObjectGuid UnitGuid;
+        
+        };
+
+        class PlayerCliGossipSelectOption final : public ClientPacket
```

Comment:
We use JAM names without prefixes. That means the "PlayerCli" part should be removed